### ISSUE TITLE
Why finetuning pika effect on CogvideoX significant better than Wan 2.1

### ISSUE THREAD

**Issue by guyuchao** [a-r-r-o-w/finetrainers#380]
Hi, I found your finetuned checkpoint on Hugging Face and the Pika Effect finetune on CogVideoX much better than Wan2.1.

Do you have any idea about the reason？

**Comment by a-r-r-o-w** [comment]
Are you finetuning 1.3B or 14B model? It trains very well on the 14B model but may be hard to make it work on 1.3B model because it is more prone to representation collapse.

Additionally, you can try to add prior preservation loss, which should stabilize lora training better. For faster convergence you can try LoKr instead of LoRA.

On the 14B model, effects can be learnt nicely with 1000-4000 steps.

**Comment by guyuchao** [comment]
I am finetuning on 1.3B. What is the meaning about "representation collapse"? Thanks for your reply.

**Comment by guyuchao** [comment]
Could you share some result video on wan 14B for reference?

**Comment by a-r-r-o-w** [comment]
Since 1.3B model is very small for video generation task and likely distilled from the 14B model as a teacher, it does not hold a lot of information and can only generate good/best quality videos when prompted in a certain way. Trying to train the model with custom data makes it go out of distribution very easily (prior preservation loss will help). I don't have many public checkpoints to share at the moment but you can take a look at these:
- https://huggingface.co/finetrainers/Wan2.1-T2V-1.3B-3dgs-v0
- https://huggingface.co/finetrainers/Wan2.1-I2V-14B-480P-3dgs-v0

The 14B model is easier to train on newer effects in my experience and can quickly learn new motion/characters/style.

The 1.3B checkpoint was trained for 5000 steps (hyperparams were not tuned and I made a mistake causing it to overfit [it should have been 500 steps]), and 14B checkpoint was trained for 1000 steps (hyperparams were not tuned). To get the intended effect in 1.3B after training, you have to use a low lora alpha (otherwise you'll get black videos due to overfitting).

**Comment by guyuchao** [comment]
I see, thanks. By the way, you mention a mistake in 1.3B training, may I know what it is?

**Comment by a-r-r-o-w** [comment]
The mistake was that: For LoRA training on a specify style/motion/character/anything else, 500-2000 steps is usually more than enough. I trained the 1.3B model for 5000 steps (the checkpoint linked above) which overfit to the data (this is not necessarily bad, but in this case it is due to the kind of dataset that was being used).

Due to this, at lora_alpha=1, the model simply generates black videos and you have the lower the lower_alpha to something like 0.5 to make the intended effect work.

The black videos are due to the nature of the dataset (all videos being mostly black leading to model simply learning to generate black videos to minimize loss): https://huggingface.co/datasets/finetrainers/3dgs-dissolve

**Comment by guyuchao** [comment]
thanks
